etc/pam.d/yubico

```diff
@@ -0,0 +1 @@
+auth [success=done default=ignore] pam_exec.so expose_authtok quiet /usr/bin/yk-auth AESKEY PASSWORD-HASH
```

Comment:
The version of the script in this repo get the key/password-hash from config file, not from arguments. So arguments are useless here.